UnityProject/Assets/Scripts/Variable Viewer/VariableViewerNetworking.cs

```diff
@@ -312,7 +308,7 @@ public int GetSize()
 				{
 					HeldBySentenceID = bob.Sentences.SentenceID
 				};
-				FriendlySentence.OnPageID = 8888888;
+				FriendlySentence.OnPageID = 8888888; //TODO?
```

Comment:
todo what?

Reply:
idk, I think this prevents you from editing lists within lists

Reply:
this will confuse people a bit in the future